mods/ra2/rules/soviet-vehicles.yaml

```diff
@@ -254,6 +254,54 @@ apoc:
 	ProducibleWithLevel:
 		Prerequisites: warfactory.infiltrated
 
+ttnk:
+	Inherits: ^Vehicle
+	Valued:
+		Cost: 1200
+	Tooltip:
+		Name: Tesla Tank
+		Description: Soviet Russian tank armed with dual small Tesla Coils.\n  Strong vs Vehicles, Infantry\n  Weak vs Aircraft
```

Comment:
`Soviet Russian`? I think that is kinda duplicating... How about just `Russian`?